### PR TITLE
Web api for scraper

### PR DESCRIPTION
**Pull request by brianconnoly**
Initial commit with NestJS web UI for running snscraper

### CODE REVIEW

File: src/app.service.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AppService {
+  getHello(): string {
+    return 'Hello World!';
+  }
```
Comment (caffellatte) [inline src/app.service.ts]: It can be removed no longer needed

File: src/snscrape.service.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@nestjs/common';
+import { exec } from 'child_process';
```
Comment (caffellatte) [inline src/snscrape.service.ts]: Not used

File: src/app.service.ts (hunk 1 above)
Comment (brianconnoly) [reply src/app.service.ts]: Removed

File: src/snscrape.service.ts (hunk 2 above)
Comment (brianconnoly) [reply src/snscrape.service.ts]: Deleted

**Review by caffellatte — APPROVED** [review]

**Pull request merged**